Dockerfile

```diff
@@ -90,7 +90,6 @@ ENV PATH=/miniconda/envs/sqlflow-dev/bin:$PATH
 # Prepare sample datasets
 COPY example/datasets/popularize_churn.sql /docker-entrypoint-initdb.d/popularize_churn.sql
 COPY example/datasets/popularize_iris.sql /docker-entrypoint-initdb.d/popularize_iris.sql
-COPY example/datasets/create_model_db.sql /docker-entrypoint-initdb.d/create_model_db.sql
```

Comment:
This `.sql` is needed for SQLFlow quickstart. We need to `CREATE DATABASE IF NOT EXISTS sqlflow_models;` in advance so that `sqlfs` can save a model into `sqlflow_models.my_dnn_model`.

Reply:
Added back, maybe unify these later in another PR.